doc/sphinx/src/contributing.rst

```diff
@@ -213,7 +215,30 @@ Note the placement of commas and angle brackets. This example excludes
 Step 3: Create tests for your EOS
 `````````````````````````````````
 
-**Important:** this is a sublty that highlights the importance of unit tests!
+The first step is to create a new ``.cpp`` file for testing your new EOS using
+the `Catch2 <Catch2_>`_ framework to design your test. We make use of the
+Behavior Driven Development (BDD) style for Catch2, so we recommend you do the
+same. In general, we recommend you copy the general structure of one of the
+existing EOS-specific unit tests.
+
+After creating your tests, you will need to include the ``.cpp`` for your new
+test in the ``CMakeLists.txt`` file, 
+
+.. code-block cmake
+
+    add_executable(eos_unit_tests
+        catch2_define.cpp
+        eos_unit_test_helpers.hpp
+        test_eos_unit.cpp
+        test_eos_gruneisen.cpp
+        test_eos_vinet.cpp
+        test_my_new_eos.cpp
```

Comment:
Adding a nitpicky comment, just for the sake of commenting. Is there a way to boldface this so that it gets highlighted?

Reply:
I don't think you can boldface the code snippets themselves.

As an FYI @gopsub , we build the documentation for every branch in the remote repo so you can actually see these changes live:
https://lanl.github.io/singularity-eos/jhp/update_contributing_doc/src/contributing.html#step-3-create-tests-for-your-eos

However, it appears that I didn't specify the right syntax highlighting so this block didn't render. I'll fix that